bdb_transaction_cli/cli.py

```diff
@@ -1,15 +1,132 @@
-# -*- coding: utf-8 -*-
-
 import click
+import json
```

Comment:
We try to follow PEP 8, and for imports it states the following

> Imports should be grouped in the following order:
>
> * standard library imports
> * related third party imports
> * local application/library specific imports
>
> You should put a blank line between each group of imports. 

ref: https://www.python.org/dev/peps/pep-0008/#imports